specs/commandline/inspect.md

```diff
@@ -0,0 +1,271 @@
+            ├──io.wabbit-networks.buildId: 123                              //user defined payload annotations.
+      ├──unsignedAttributes
+            ├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>  //how TSA response (time stamp token) is represented.
+            ├──io.cncf.notary.signingAgent: notation/1.0.0                  //provides the identifier of a client (e.g. Notation) that produced the signature
+      ├──certificates
+            ├──SHA1Fingerprint: 2f1cc5b8455381cdefac83b4bd305b789cc9c16e
+              ├──issuedTo: Microsoft Root Certificate Authority 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sat Jun 23 22:04:01 2035
+            ├──SHA1Fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+              ├──issuedTo: Microsoft Code Signing PCA 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sun Jul 06 20:50:17 2025
+      └──payload                                                           //descriptor of the target artifact manifest that is signed.
+            ├──mediaType: application/vnd.oci.image.manifest.v1+json
+            ├──digest: sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+            └──size: 16724
+      
+    ├── sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+      ├──signingAlgorithm: RSASSA-PSS-SHA-256
+      ├──signedAttributes
+            ├──contentType: application/vnd.cncf.notary.payload.v1+json
+            ├──signingScheme: notary.signingAuthority.x509
+            ├──signingTime: 2022-04-06T07:01:20Z
+            ├──expiry: 2022-10-06T07:01:20Z
+            ├──io.cncf.notary.verificationPlugin: com.example.nv2plugin                  
+      ├──unsignedAttributes
+            ├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>
+            ├──io.cncf.notary.signingAgent: notation/1.0.0                  
+      ├──certificates
+            ├──SHA1Fingerprint: 2f1rr5b8455381frdajc83b4bd305b743cc9513u
+              ├──issuedTo: Microsoft Root Certificate Authority 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sat Jun 23 22:04:01 2035
+            ├──SHA1Fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+              ├──issuedTo: Microsoft Code Signing PCA 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sun Jul 06 20:50:17 2025
+      └──payload 
+            ├──mediaType: application/vnd.oci.image.manifest.v1+json
+            ├──digest: sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+            └──size: 16724
+```
+
+## Usage signatures on an OCI artifact identified by a tag
+
+```text
+`Tags` are mutable, but `Digests` uniquely and immutably identify an artifact. If a tag is used to identify a signed artifact, notation resolves the tag to the `digest` first.
```

Comment:
Please remove the text block as it is not necessary if you use plain text in the Markdown.